src/utils.js

```diff
@@ -11,18 +11,29 @@ export const listenerOptions = {
 
 export function preventTouchMove(e) {
   e.preventDefault();
-  
+
   return false;
 }
 
 export function allowTouchMove(e) {
   const target = e.currentTarget;
-  
+
   if (target.scrollHeight > target.clientHeight) {
     e.stopPropagation();
     return true;
   }
 
+  var horizontalScroll = target.querySelector(
+    'div[data-horizontal-scroll="true"]',
```

Comment:
This (`data-horizontal-scroll`) isn't referenced anywhere else in the PR, is it part of the proposed API?